### PR TITLE
Design implementation fixes

### PR DESCRIPTION
**Pull request by Royal-lobster**
Closes https://github.com/EveripediaNetwork/issues/issues/594

# Changes
- changed description color to whiteAlpha500 in homepage dark mode
- fixed wallet drawer button color, font weights and spacings to match design
- fixed spacing between nav items in desktop mode
- changed hero text to a dual tone colors
- changed the font of site to have montserrat by default
- made category card images to take path from url for faster loading
- fixed search bar overflowing in some cases including when a wiki have more than 3 tags
- other smaller fixes.

# Links
- https://monopedia-ui-git-design-syncs-30-prediqt.vercel.app/

### CODE REVIEW

**Comment by kesar** [comment]
coming back to the hero buttons @Simplypheyie96 what if we keep Explore (perhaps another name?) as primary action and Create we change it for Learn More and remove the link? I think we have create in navigation and it shouldnt be an action a normal user can perform when they hit the landing

**Comment by Simplypheyie96** [comment]
> coming back to the hero buttons @Simplypheyie96 what if we keep Explore (perhaps another name?) as primary action and Create we change it for Learn More and remove the link? I think we have create in navigation and it shouldnt be an action a normal user can perform when they hit the landing

Totally on point. Explore just fits the purpose of the site perfectly, so I think it is fine. 
image

PS: I Have a little deal with some changes on figma, but I will work on the copy by tomorrow. I need to inspect all of the files on everipedia and IQ dashboard and others to set things right.

**Comment by kesar** [comment]
lets try to get this merged before call so we show some updates regarding landing and page. If more changes are needed we can add a new PR later.

**Review by kesar — APPROVED** [review]

**Comment by Simplypheyie96** [comment]
image
@Royal-lobster this should be learn more

**Review by Simplypheyie96 — CHANGES_REQUESTED** [review]
image
@Royal-lobster this should be learn more

image
the help center should be "IQ Learn"
image
wrong colors here in dark mode. and wrong pink in light and dark mode
correct pink for "wallet" in light mode is brand 500
correct text color and pink in dark mode is alphaa-white 500/ Brand 800

File: src/utils/i18n.js [hunk 1]
```diff
@@ -336,11 +332,9 @@ export const resources = {
       // init
       //Home page
       everipedia: 'Everipedia',
-      iq_description: 'Everipedia is the world’s largest crypto knowledge base',
-      hero_title: 'An Ecosystem of Knowledge on the Blockchain.',
+      iq_description: 'Everipedia is the biggest Blockchain Encyclopedia',
       exploreHeroBttn: 'Explore',
-      createHeroBttn: 'Create',
-      learnMoreHeroBttn: 'Read more about Everipedia',
+      learnMoreHeroBttn: 'Learn',
```
Comment (Royal-lobster) [inline src/utils/i18n.js:337]:
```suggestion
      learnMoreHeroBttn: 'Learn More',
```

File: src/utils/i18n.js [hunk 2]
```diff
@@ -193,11 +191,9 @@ export const resources = {
 
       //Home page
       everipedia: 'Everipedia',
-      iq_description: 'Everipedia is the world’s largest crypto knowledge base',
-      hero_title: 'An Ecosystem of Knowledge on the Blockchain.',
+      iq_description: 'Everipedia is the biggest Blockchain Encyclopedia',
       exploreHeroBttn: 'Explore',
-      createHeroBttn: 'Create',
-      learnMoreHeroBttn: 'Read more about Everipedia',
+      learnMoreHeroBttn: 'Learn',
```
Comment (Royal-lobster) [inline src/utils/i18n.js:196]:
```suggestion
      learnMoreHeroBttn: 'Learn More',
```

File: src/utils/i18n.js [hunk 3]
```diff
@@ -11,11 +11,9 @@ export const resources = {
...
```
Comment (Royal-lobster) [inline src/utils/i18n.js:16]:
```suggestion
      learnMoreHeroBttn: 'Learn More',
```

**Review by kesar — APPROVED** [review]

**Pull request merged**